bundles/core/org.eclipse.smarthome.core.thing/src/main/java/org/eclipse/smarthome/core/thing/firmware/ProgressCallbackImpl.java

```diff
@@ -34,6 +34,7 @@
  *
  * @author Thomas Höfer - Initial contribution
  * @author Christoph Knauf - Introduced pending, canceled, update and InternalState
+ * @author Dimitar Ivanov - Callback contains firmware domain object
```

Comment:
Despite it is package-private, could you please move it to `org.eclipse.smarthome.core.thing.internal.firmware.ProgressCallbackImpl`?